modules/pubmaticServerBidAdapter.js

```diff
@@ -145,20 +139,11 @@ function _createImpressionObject(bid, conf) {
     ext: {
       pmZoneId: _parseSlotParam('pmzoneid', bid.params.pmzoneid),
       div: bid.params.divId,
-      adunit: bid.params.adUnitId,
-      slotIndex: bid.params.adUnitIndex
+      adunit: (bid.params.adUnitId.split(':')[0]) // Removing :XX structure if any found
     }
   };
 }
 
-function mandatoryParamCheck(paramName, paramValue) {
```

Comment:
why this function is removed ? we wanted to put meaningful log upon when a mandatory param is missing

Reply:
hasValidBidRequest still do the same thing & put the logs